### PR TITLE
📝 Add draft architecture diagram

### LINKED ISSUE #1: First draft architecture diagram for FHIR based clinical data pipeline

### PR DESCRIPTION
**Pull request by znatty22**
Closes #1

### CODE REVIEW

**Comment by allisonheath** [comment]
Looks reasonable - I'd just be a bit more agnostic about the tool used from REDCap->FHIR at this point as I think there will be a few different methods emerging over this year?

File: README.md [hunk 1]
```diff
@@ -7,3 +8,16 @@
 
 # Clinical Data Flow
 Project management and design artifacts for [Clinical Data Flow](https://d3b.io/docs/products/clinical-data-flow) product
+
+## Evolving Architecture
+[Draw.io Source](https://drive.google.com/file/d/1xdtljwZ2FjFVxJD4tPRbP1QWk2j1eE3d/view?usp=sharing)
```
Comment (znatty22) [inline README.md]: Add KF dataservice into diagram

File: README.md [hunk 2]
```diff
@@ -7,3 +8,16 @@
...
+
+![Architecture](docs/static/img/clinical-data-flow-draft-arch.png)
+
+### Components
```
Comment (znatty22) [inline README.md]: Add link to KF dataservice into diagram

File: README.md (hunk 1 above)
Comment (liberaliscomputing) [reply README.md]: In the long run, we also want to persist records from REDCap into the CDF warehouse, right?

**Pull request merged**